### PR TITLE
Remove Eternatus-Eternamax

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: sim/team-validator.ts [hunk 1]
```diff
@@ -1186,7 +1186,7 @@ export class TeamValidator {
 		}
 
 		if (
-			tierTemplate.isNonstandard === 'Pokestar' && dex.gen === 5 ||
+			tierTemplate.isNonstandard === 'Unusable' && dex.gen === 5 ||
```
Comment (TheImmortal) [inline sim/team-validator.ts]: Eternatus isn't gen 5

Comment (KrisXV) [reply sim/team-validator.ts]: fixed

File: sim/team-validator.ts [hunk 2]
```diff
@@ -1186,7 +1186,7 @@ export class TeamValidator {
 		}
 
 		if (
-			tierTemplate.isNonstandard === 'Pokestar' && dex.gen === 5 ||
```
Comment (xfix) [inline sim/team-validator.ts]: This allows using Pokestar mons in Gen6.

Comment (KrisXV) [reply sim/team-validator.ts]: done i think

**Pull request merged**